pkg/host/syscalls_linux.go

```diff
@@ -260,6 +260,13 @@ func isSyzFuseSupported(c *prog.Syscall, target *prog.Target, sandbox string) (b
 	return true, ""
 }
 
+func isSyzUsbIpSupported(c *prog.Syscall, target *prog.Target, sandbox string) (bool, string) {
+	//if err := osutil.IsAccessible("/sys/devices/platform/vhci_hcd.0/attach"); err != nil {
```

Comment:
This attach file is only accessible by the root. So, when I include this line, all the relevant syscalls are disabled by syzkaller.

Reply:
This code runs as root, so it should work. Maybe osutil.IsAccessible is trying to do something that's not permitted on this file?

But then syz_usbip_server_init won't legitimately be able to open it, because syscalls don't always run as root (depends on sandbox parameter). So you also need to check sandbox parameter here.

Question: does opening /sys/devices/platform/vhci_hcd.0/attach creates a new server each time? Or it attaches to the same single server?

Reply:
Re `IsAccessible`: This file is only openable for writing, looks like this is the reason `IsAccessible` is failing.

```
# ls -al /sys/devices/platform/vhci_hcd.0/attach
--w-------. 1 root root 4096 Sep 16 20:32 /sys/devices/platform/vhci_hcd.0/attach
```

Perhaps we should add `IsWritable` or something?

Reply:
Yes, it makes sense. isWritable can work properly. 

> Question: does opening /sys/devices/platform/vhci_hcd.0/attach creates a new server each time? Or it attaches to the same single server?
No, there may be different clients who may attach the USB devices of the same single server.  This "attach" file is used for USB/IP communication. For each USB/IP connection, another port is used, therefore another vhci_hcd. A new attach file is created for different devices or clients.

Reply:
If we add IsWritable, it will make IsAccessible name quite confusing and will open door for introducing bugs by using a wrong function. It will be better to have IsWritable, IsReadable and IsReadWritable set so that it's clear in the existing code and whoever adds new calls will need to think what exactly they are asking for.